admin/releasing.md

```diff
@@ -0,0 +1,68 @@
+Releasing a Log4cxx version
+===================
+
+This document lists the steps that must be performed to release Log4cxx
+using 1.3.0 as the example.
+
+Prerequisites
+----------
+
+* The version number (in src/cmake/projectVersionDetails.cmake) has been updated
+* The change log (in src/site/markdown/change-report-gh.md) is up-to-date.
+* The web-site for the new version has been published to https://logging.staged.apache.org/log4cxx
+* An e-mail has been sent to dev@logging.apache.org announcing the intention to release
+* Your public key is available in [Apache Logging KEYS file](https://dist.apache.org/repos/dist/release/logging/KEYS)
+
+Steps
+-----
+
+1. Tag HEAD as the release candidate
+    - `git checkout master`
+    - `git push origin tag v1.3.0-RC1`
+1. Remove the old artifacts from svn
+    - `svn co https://dist.apache.org/repos/dist/dev/logging -N apache-dist-logging-dev`
+    - `cd apache-dist-logging-dev`
+    - `svn up log4cxx`
+    - `cd log4cxx`
+    - `svn delete *`
+1. Download the latest release artifacts
+    - The link to the packaged artifacts is available
+      in the Github action log under the "Run action/upload-artifact" step of
+      the "Generate release files" Github action.
+      The log of the most recent commit can be accessed
+      using the "Details" link in the pop-up window shown
+      when the green tick is clicked.
+    - `cd apache-dist-logging-dev/log4cxx`
+    - `unzip "~/Downloads/Upload release files.zip"`
+1. Sign release artifacts (Refer: https://infra.apache.org/release-signing.html)
```

Comment:
These two steps here(`Download the latest release artifacts` and `Sign release artifacts`) should be automatically handled for you if you do `-DAPACHE_MAINTAINER=ON`; I know you changed something about that recently and I haven't gone back and checked to make sure that it still works.

Reply:
[INFRA-26194](https://issues.apache.org/jira/browse/INFRA-26194) added secrets to our runners, so we can do those things automatically, i.e.:

1. Sign the archives on the Github runner.
2. Commit everything to the `dev` subversion repository.

I can try updating the workflows tomorrow.